app/public/service-worker.js

```diff
@@ -0,0 +1,90 @@
+var reviewRedirect, title, options;
+title = 'We Need Your Help!';
+options = {
+	body: '',
+	icon: '/images/logo-192x192.png',
+	sound: '/sounds/alert.wav',
```

Comment:
Did you forget to commit this public/sounds/alert.wav file?

Also, is it big?  mp3 may be better compressed.

Reply:
I must've lost it when I was fixing rebase issues, I'll add it back and change it to an mp3